src/project/SidebarView.js

```diff
@@ -54,12 +54,12 @@ define(function (require, exports, module) {
         $projectTitle,
         $projectFilesContainer,
         $workingSetViewsContainer;
-
     var _cmdSplitNone,
         _cmdSplitVertical,
         _cmdSplitHorizontal;
+    var MinimumSidebarWidthPX = 200;
 
-    /**
+     /**
```

Comment:
this indent should not be here. The format is:
```
/**
 * ...
 */
```
So that the asterisks (the "*" symbols) all line up.